github/ContentFile.py

```diff
@@ -93,6 +93,18 @@ def html_url(self):
         self._completeIfNotSet(self._html_url)
         return self._html_url.value
 
+    @property
+    def last_modified(self):
+        """
+        :type: str
```

Comment:
Maybe datetime?

Reply:
Actually I did start off by using `datetime`, but figured it would be better to stay consistent because AFAICS all other objects will return string for `last_modified.